lib/services/settings.dart

```diff
@@ -37,14 +37,6 @@ class Settings {
     _set('logWrap', enabled);
   }
 
-  String _getString(String key, String defaultValue) {
```

Comment:
Any reason to remove this?

Reply:
It wasn't being used, and gave a linter error/warning.

Reply:
I think even though it's not being used currently, it's worth keeping around, even if a dead-code linter is firing. I don't think it's far from the realm of possibilities that we'll want to store a string value from the settings in the future.